contracts/Manager.sol

```diff
@@ -31,15 +34,15 @@ contract Manager is IManager {
         _;
     }
 
-    constructor(address _controller) public {
+    constructor(address _controller) {
         controller = IController(_controller);
     }
 
     /**
      * @notice Set controller. Only callable by current controller
      * @param _controller Controller contract address
      */
-    function setController(address _controller) external onlyController {
+    function setController(address _controller) external override onlyController {
```

Comment:
we don't need to override

Reply:
Since this function is also defined in the `IManager` interface, which this contract inherits we need to override